AccountingSystem/Client/DataStoreClient.py

```diff
@@ -1,86 +1,81 @@
 """ Module that holds the DataStore Client class
 """
 
+__RCSID__ = "$Id$"
+
 import time
 import random
 import copy
 import threading
 
 from DIRAC import S_OK, S_ERROR, gLogger, gConfig
-from DIRAC.Core.DISET.RPCClient                     import RPCClient
-from DIRAC.Core.Utilities                           import DEncode
-from DIRAC.RequestManagementSystem.Client.Request   import Request
+from DIRAC.Core.Base.Client import Client
+from DIRAC.Core.Utilities import DEncode
+from DIRAC.RequestManagementSystem.Client.Request import Request
 from DIRAC.RequestManagementSystem.Client.Operation import Operation
 from DIRAC.RequestManagementSystem.Client.ReqClient import ReqClient
 from DIRAC.Core.Utilities.DErrno import ERMSUKN
 
-__RCSID__ = "$Id$"
-
 random.seed()
 
-class DataStoreClient(object):
+
+class DataStoreClient(Client):
   """
     Class providing front end access to DIRAC Accounting DataStore Service
      - It allows to reduce the interactions with the server by building and list of
     pending Registers to be sent that are sent in a bundle using the commit method.
      - In case the DataStore is down Registers are sent as DISET requests.
   """
-  def __init__( self, setup = False, retryGraceTime = 0 ):
-    self.__setup = setup
```

Comment:
What was this setup thing ? is it not needed anymore ?

Reply:
Something very (very!) old that's certainly not needed anymore.